airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -14,53 +14,105 @@
 from typing_extensions import Literal
 
 
-class AddedFieldDefinition(BaseModel):
+class DefinitionOfFieldToAdd(BaseModel):
```

Comment:
@brianjlai is it expected that setting a title changes the class name? I'm not super worried since these classes are only used internally, but it's something to keep in mind.